### DISCUSSION TITLE
the graphic is broken when using MVTLayer

### DISCUSSION THREAD

**Discussion by lizhikang555** [visgl/deck.gl#6717] [Q&A] [answered]
The graphic is broken when I use MVTLayer and don't set the value of 'binary'
![image](https://user-images.githubusercontent.com/51242828/155964020-225b9b25-8e71-462e-96c8-5037ebfed58e.png)
my code:
```
const simpleMeshLayer = new SimpleMeshLayer({
    id: 'earth-sphere',
    data: [0],
    mesh: new SphereGeometry({ radius: EARTH_RADIUS_METERS, nlat: 18, nlong: 36 }),
    coordinateSystem: COORDINATE_SYSTEM.CARTESIAN,
    getPosition: [0, 0, 0],
    getColor: [255, 255, 255],
    getPolygonOffset: p => [0,0]
})

const countryLayer = new MVTLayer({
    id: 'Country',
    // binary:false,
    data: `http://localhost:8090/hub/spaces/Country/tile/web/{z}_{x}_{y}.mvt`,
    minZoom: 0,
    maxZoom: 23,
    getLineColor: [204, 255, 255],
    getFillColor: [250, 128, 114],
    getLineWidth: 1,
    lineWidthMinPixels: 1,
    getPolygonOffset: p => [0, -100]
})
```
---------------------------------------------------------------------------------------------------------------------------------------------------
but confusingly,when I only set 'binary' to 'false' without changing anything else,the graphic is displayed normally
![image](https://user-images.githubusercontent.com/51242828/155964937-1bf345b7-2c81-45a9-845d-36d013f8f223.png)
my code:
```
const countryLayer = new MVTLayer({
    id: 'Country',
    binary:false,
    data: `http://localhost:8090/hub/spaces/Country/tile/web/{z}_{x}_{y}.mvt`,
    minZoom: 0,
    maxZoom: 23,
    getLineColor: [204, 255, 255],
    getFillColor: [250, 128, 114],
    getLineWidth: 1,
    lineWidthMinPixels: 1,
    getPolygonOffset: p => [0, -100]
})
```
I don't know what the problem is.Is there a problem with my MVT data?Or I have to set 'binary' to 'false'?

**Comment by Pessimistress** [comment, marked as answer]
I do not believe `binary:true` mode supports the GlobeView. @felixpalmer should the prop default to false in GlobeView?

**Reply by felixpalmer** [reply]
I would say yes, given that GlobeView is marked as experimental it is simpler change the default for now. I will open a PR for this
